client/homebrew/homebrew.jsx

```diff
@@ -42,15 +41,32 @@ const Homebrew = (props)=>{
 		brews
 	} = props;
 
+	const [isClient, setIsClient] = useState(false);
+
+	useEffect(()=>{
+		setIsClient(true);
+	}, []);
+
```

Comment:
Two comments related to `isClient`:

1) Can we just use `typeof window !== undefined` instead of this whole state/useEffect thing?
2) Why do we care if we in a client?

Reply:
Ok. Just looking at the diff, `isClient` did not exist at all before this PR, and there was no check for being in the client; we were just relying on the config to tell us if we were in a deployment environment. I would assume we are still safe to just use the config environment without introducing a new state or a check for being in the client.